### DISCUSSION TITLE
svelte integration?

### DISCUSSION THREAD

**Discussion by baseplate-admin** [scalar/scalar#1569] [Ideas]
Hi, it seems that there's no integration for svelte. Do you guys plan on adding svelte support?

**Comment by marclave** [comment]
hi @baseplate-admin ! not on our roadmap yet but we should add it :)

**Reply by baseplate-admin** [reply]
Yep Now that there's this 

- https://github.com/scalar/scalar/pull/1748

I think it's better to add to roadmap
